android/src/main/java/com/atroo/securestorage/SecureStoragePluginPlugin.java

```diff
@@ -0,0 +1,22 @@
+package com.atroo.securestorage;
+
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+import com.getcapacitor.annotation.CapacitorPlugin;
+
+@CapacitorPlugin(name = "SecureStoragePlugin")
+public class SecureStoragePluginPlugin extends Plugin {
```

Comment:
Same remark regarding Wording like for iOS. Do not use PluginPlugin

And package should be de.atroo not com.atroo :)

Reply:
OK